http/src/main/java/io/micronaut/http/form/FormConfiguration.java

```diff
@@ -26,4 +26,10 @@ public interface FormConfiguration {
      * @return default maximum of decoded key value parameters. It defaults to 1024.
      */
     int getMaxDecodedKeyValueParameters();
+
+    /**
+     * @return true if the semicolon handle as a normal character, false otherwise.
+     */
+    boolean isSemicolonIsNormalChar();
```

Comment:
to avoid a breaking change this should be a default method that returns `false`

Reply:
fixed

Reply:
fixed